web/preferences.js

```diff
@@ -49,24 +49,18 @@ function cloneObj(obj) {
 }
 
 /**
- * Preferences - Utility for storing persistent settings.
+ * BasePreferences - Base class for storing persistent settings.
  *   Used for settings that should be applied to all opened documents,
  *   or every time the viewer is loaded.
  */
-var Preferences = {
-  prefs: null,
-  isInitializedPromiseResolved: false,
```

Comment:
Note that was simply removed, since there wasn't a single place in the code that actually used it.